web/src/MainLayout.jsx

```diff
@@ -69,6 +70,21 @@ const Header = () => {
   );
 };
 
+const ChangeProductButton = () => {
```

Comment:
Thanks!

I actually meant to put only the button and it's logic in its own file somewhere, without the PageSidebarBody. The idea is that such a button could be mounted in other places if needed.

But don't worry by now and keep it as it is now. There still a lot work to do regarding the UI. I can relocate it later.